controllers/comments.controller.js

```diff
@@ -9,3 +9,18 @@ exports.getCommentsByArticleId = (req, res, next) => {
         })
         .catch(next);
 };
+
+exports.postCommentByArticleId = (req, res, next) => {
+    const { article_id } = req.params;
+    const { username, body } = req.body;
+  
+    if (!username || !body) {
+      return res.status(400).send({ msg: 'Missing required fields: username or body' });
+    }
+  
```

Comment:
You don't need to manually check the above - you can let psql deal with it and throw an error itself. If these values are missing it violates the rules set in the seed. You could catch an error and then use the code to send a response.